mlflow/tracking/Dockerfile

```
@@ -0,0 +1,7 @@
+# example build command docker build . --build-arg TAG=v1.28.0
+FROM python:3.10.6-slim-bullseye
+ARG TAG
+RUN echo $TAG 
+RUN apt-get update && apt-get install git -y && \
+	pip install git+https://github.com/mlflow/mlflow.git@$TAG
```

```suggestion
	pip install --no-cache git+https://github.com/mlflow/mlflow.git@$TAG
```

to reduce the image size.

---

addressed with Pypi commit